app/assets/stylesheets/modules/_wizard.styl

```diff
@@ -193,12 +193,18 @@ wizard-width = 860px
         border-top 1px solid transparent
         box-sizing content-box
         overflow hidden
-        display none
-        margin-top 18px
-        padding 15px 0 0
+        height 0px
         width 100%
+        transition all .2s ease-in-out
+        opacity 0
         &.open
           border-color mischka
+      .wizard__option__description.open
+        height 100%
+        margin-top 18px
+        padding 15px 0 0
+        opacity 1
```

Comment:
What are these changes for?

Reply:
This used to use JQuery's `slideToggle` to implement the closing and opening animation. This doesn't have a direct alternative in JS so I used some css animations instead.